src/assets/data/defaultConfig.ts

```diff
@@ -134,22 +131,39 @@ export const DefaultEndPoint: string =
   'https://radar-cns-platform.rosalind.kcl.ac.uk/'
 // export const DefaultEndPoint: string = 'https://radar-backend.co.uk/'
 
-export const DefaultSchemaEndpoint =
-  'https://raw.githubusercontent.com/RADAR-base/RADAR-Schemas/add-audio/'
-export const DefaultSchemaSpecExtension = '.yml'
-export const DefaultSchemaSpecEndpoint =
-  DefaultSchemaEndpoint +
-  'specifications/active/aRMT-' +
-  DefaultSourceTypeRegistrationBody.sourceTypeCatalogVersion +
-  DefaultSchemaSpecExtension
-
-export const DefaultProtocolEndPoint: string = DefaultProtocolEndPointExport
-
-export const DefaultSourceProducerAndSecret: string = DefaultSourceProducerAndSecretExport
+// GITHUB SOURCES
+
+export const GIT_API_URI = 'https://api.github.com/repos'
+export const DefaultProject = 'RADAR-Base'
```

Comment:
Perhaps name it DefaultOwner or DefaultOrganisation. `Project` can be ambiguous with a study in radar